docs/tutorials/intro.md

```diff
@@ -209,8 +238,41 @@ fn main() {
 }
 ```
 
+You will also need to use the `with_buffer_provider` constructors, e.g. `DateTimeFormatter::try_new_with_buffer_provider()`.
+
 The ICU4X repository has test data checked in tree in `provider/testdata/data`, however it is recommended one generate data on their own as described in the [next section](#generating-data). Under the hood, `icu_testdata` is simply loading this data.
```

Comment:
"checked in tree" isn't relevant to the user because they have not checked out our tree.
```suggestion
```

Reply:
I changed it to mention checking it out